Firmware/RTK_Everywhere/PointPerfectLibrary.ino

```diff
@@ -33,15 +33,27 @@ void updatePplTask(void *e)
             {
                 if (rtcmLength > 0)
                 {
-                    if (settings.debugCorrections == true)
-                        systemPrintln("Received RTCM from PPL. Pushing to the GNSS.");
-
-                    gnssPushRawData(pplRtcmBuffer, rtcmLength);
+                    // You could imagine a scenario where you feed the data from a NEO-D9S into the PPL,
+                    // because you believe the PPL can do a better job generating corrections than the ZED.
+                    // TODO: check if we want to add a CORR_LBAND_PPL corrections source - for the ZED
+                    updateCorrectionsLastSeen(CORR_LBAND);
+                    if (isHighestRegisteredCorrectionsSource(CORR_LBAND))
```

Comment:
So you're routing all SPARTN through the PPL regardless of the source of the SPARTN? (Could be NEO, could be IP PointPerfect). I guess I'm ok with that but I suspect the PPL will trail the ZED firmware in its abilities/features. If/when new ZED firmware lands this summer, we may need the CORR_LBAND_PPL ability.